doc/README.md

```diff
@@ -41,9 +41,10 @@ The following are developer notes on how to build Bitcoin Core on your native pl
 - [macOS Build Notes](build-osx.md)
 - [Unix Build Notes](build-unix.md)
 - [Windows Build Notes](build-windows.md)
+- [FreeBSD Build Notes](build-freebsd.md)
 - [OpenBSD Build Notes](build-openbsd.md)
 - [NetBSD Build Notes](build-netbsd.md)
-- [Gitian Building Guide](gitian-building.md)
+- [Gitian Building Guide (External Link)](https://github.com/bitcoin-core/docs/blob/master/gitian-building.md)
```

Comment:
I don't understand this change, why change this to an external link?

Reply:
I think I forgot to adjust this when I moved the file to the new docs repository

Reply:
Yea, it's just to save a hop to `gitian-building.md` in this repo, which just redirects you to `https://github.com/bitcoin-core/docs/blob/master/gitian-building.md` anyways.

Reply:
OH, it refers to a different file, hadn't noticed